### PR TITLE
remove service status "feature"

### PR DESCRIPTION
**Pull request by ebdavison**
Per the issue tracker, this was removed in issue Mailu#463 (Remove the Service Status page)

## What type of PR?

documentation

## What does this PR do?

remove feature for services status which no longer exists; this confused me as I was trying to find it and was not able to.

## Related issue(s)

Remove the Service Status page Mailu#463

## Prerequistes

Before we can consider review and merge, please make sure the following list is done and checked.
If an entry in not applicable, you can check it or remove it from the list.

N/A

### CODE REVIEW

**Comment by ofthesun9** [comment]
Good catch ! Thanks for the PR

**Review by ofthesun9 — APPROVED** [review]

File: docs/features.rst [hunk 1]
```diff
@@ -16,10 +16,6 @@ Admin interface screenshots
 
    Managing email domains
 
-.. figure:: assets/screenshots/status.png
```
Comment (lub) [inline docs/features.rst:19]: Can you also remove the mentioned image? I don't think we need it anymore after deleting this reference.

Comment (ebdavison) [reply docs/features.rst:19]: Sure thing.  I have done that and it is now showing in this PR.

Comment (lub) [reply docs/features.rst:19]: thx :)

**Review by lub — APPROVED** [review]

**Review by ofthesun9 — APPROVED** [review]

**Comment by ofthesun9** [comment]
Bors try

**Pull request merged**